charts/incubator/lancache-dns/questions.yaml

```diff
@@ -0,0 +1,333 @@
+portals: {}
+questions:
+# Include{global}
+  - variable: controller
+    group: "Controller"
+    label: ""
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: advanced
+          label: "Show Advanced Controller Settings"
+          schema:
+            type: boolean
+            default: false
+            show_subquestions_if: true
+            subquestions:
+              - variable: type
+                description: "Please specify type of workload to deploy"
+                label: "(Advanced) Controller Type"
+                schema:
+                  type: string
+                  default: "deployment"
+                  required: true
+                  enum:
+                    - value: "deployment"
+                      description: "Deployment"
+                    - value: "statefulset"
+                      description: "Statefulset"
+                    - value: "daemonset"
+                      description: "Daemonset"
+              - variable: replicas
+                description: "Number of desired pod replicas"
+                label: "Desired Replicas"
+                schema:
+                  type: int
+                  default: 1
+                  required: true
+              - variable: strategy
+                description: "Please specify type of workload to deploy"
+                label: "(Advanced) Update Strategy"
+                schema:
+                  type: string
+                  default: "RollingUpdate"
+                  required: true
+                  enum:
+                    - value: "Recreate"
+                      description: "Recreate: Kill existing pods before creating new ones"
+                    - value: "RollingUpdate"
+                      description: "RollingUpdate: Create new pods and then kill old ones"
+                    - value: "OnDelete"
+                      description: "(Legacy) OnDelete: ignore .spec.template changes"
+# Include{controllerExpert}
+  - variable: env
+    group: "Container Configuration"
+    label: "Image Environment"
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: CACHE_DOMAINS_REPO
+          label: "CACHE DOMAINS REPO"
+          description: "The github repo url for the cache_domains files to use to populate the cdn maps."
+          schema:
+            type: string
+            default: "https://github.com/uklans/cache-domains.git"
+        - variable: CACHE_DOMAINS_BRANCH
+          label: "CACHE DOMAINS BRANCH"
+          description: "The branch of the CACHE_DOMAINS_REPO to use"
+          schema:
+            type: string
+            default: "master"
+        - variable: UPSTREAM_DNS
+          label: "UPSTREAM DNS"
+          description: "Upstream DNS server(s) the cache should use(separated by spaces). Do NOT point this at lancache-dns)"
+          schema:
+            type: string
+            default: "1.1.1.1 1.0.0.1"
```

Comment:
Technically the `;` separator is correct aswell. But I prefer ` `. Because if `;` is not parsed correctly could cause problems :)
(That's why I changed it on your previous PR

Reply:
I didnt see that and yes I should have used a spaced, ty.